openedx_events/analytics/signals.py

```diff
@@ -0,0 +1,23 @@
+"""
+Standardized signals definitions for events within the architecture subdomain ``analytics``.
+
+All signals defined in this module must follow the name and versioning
+conventions specified in OEP-41.
+
+They also must comply with the payload definition specified in
+docs/decisions/0003-events-payload.rst
+"""
+
+from openedx_events.analytics.data import TrackingLogData
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# .. event_type: org.openedx.analytics.event_tracking.emitted.v1
```

Comment:
The event_type must match the events' implemented name:

```suggestion
# .. event_type: org.openedx.analytics.tracking.event.emitted.v1
```
or 
```suggestion
# .. event_type: org.openedx.analytics.tracking_event.emitted.v1
```
Or is this some other convention for tracking events?